### PR TITLE
fix: Ensure 'mozilla-vpn-addons' release aren't filtered out of relea…

### PR DESCRIPTION
**Pull request by ahal**
…ses view

Bug: 1871138

### CODE REVIEW

**Comment by ahal** [comment]
I jumped the gun and accidentally merged https://github.com/mozilla-releng/shipit/pull/1384 before pushing up some changes that were needed to get the Addons releases to show up in the releases view.

File: startup.sh [hunk 1]
```diff
@@ -9,6 +9,8 @@ scopes=('hooks:trigger-hook:project-app-services/in-tree-action-1-generic/*' \
         'hooks:trigger-hook:project-gecko/in-tree-action-1-release-promotion/*' \
         'hooks:trigger-hook:project-mobile/in-tree-action-1-generic/*' \
         'hooks:trigger-hook:project-mobile/in-tree-action-1-release-promotion/*' \
+        'hooks:trigger-hook:project-mozillavpn/in-tree-action-1-generic/*' \
+        'hooks:trigger-hook:project-mozillavpn/in-tree-action-1-release-promotion/*' \
```
Comment (jcristau) [inline startup.sh:13]: Should this hunk be a separate commit and/or PR?

**Review by jcristau — APPROVED** [review]

File: startup.sh (hunk 1 above)
Comment (ahal) [reply startup.sh:13]: Thanks for calling me out.. I'm always preaching atomic commits :p.
